utils/graph.h

```diff
@@ -57,6 +64,9 @@ void graph_init(struct uftrace_graph *graph, struct uftrace_session *s);
 void graph_init_callbacks(graph_fn entry, graph_fn exit, graph_fn event, void *arg);
 void graph_destroy(struct uftrace_graph *graph);
 
+extern void update_graph_time_stat(struct graph_time_stat *ts, uint64_t time_n);
+extern void init_time_stat(struct graph_time_stat *ts);
```

Comment:
For external functions, it'd be good to have a common prefix like graph.  Please rename to `graph_update_time_stat` and `graph_init_time_stat`.